CollapseLauncher/XAMLs/MainApp/Pages/OOBE/OOBEStartUpMenu.xaml

```diff
@@ -0,0 +1,270 @@
+                        <TextBlock Text="{x:Bind helper:Locale.Lang._StartupPage.Subtitle3}" Style="{ThemeResource SubtitleTextBlockStyle}" FontWeight="Normal" Margin="0,0,0,32"/>
+                        <TextBlock Style="{ThemeResource SubtitleTextBlockStyle}" Margin="0,0,0,16">
+                            <Run Text="{x:Bind helper:Locale.Lang._StartupPage.Subtitle4_1}" FontWeight="Normal"/>
+                            <Run Text="{x:Bind helper:Locale.Lang._StartupPage.Subtitle4_2}" Foreground="{ThemeResource SystemAccentColor}"/>
+                            <Run Text="{x:Bind helper:Locale.Lang._StartupPage.Subtitle4_3}" FontWeight="Normal"/>
+                        </TextBlock>
+                        <StackPanel Orientation="Vertical">
+                            <TextBlock Margin="0,0,0,16" x:Name="ErrMsg" Text="" TextWrapping="Wrap" TextTrimming="Clip" VerticalAlignment="Center" HorizontalAlignment="Left" FontWeight="SemiBold" Foreground="{ThemeResource TextFillColorPrimary}"/>
+                            <Button Style="{ThemeResource AccentButtonStyle}" Click="ChooseFolder" CornerRadius="16">
+                                <StackPanel Orientation="Horizontal" Margin="8,0">
+                                    <TextBlock Text="{x:Bind helper:Locale.Lang._StartupPage.ChooseFolderBtn}" Margin="0,0,8,0" FontWeight="Medium" TextWrapping="Wrap"/>
+                                    <FontIcon FontFamily="{ThemeResource FontAwesomeSolid}" Glyph="&#xf07b;" FontSize="14"/>
+                                </StackPanel>
+                            </Button>
+                        </StackPanel>
+                    </StackPanel>
+                </ScrollViewer>
+                <ScrollViewer x:Name="CustomizationContainer" Grid.Row="1" Margin="0,0,0,24" Grid.ColumnSpan="2" Visibility="Collapsed">
+                    <StackPanel x:Name="SettingsCardContainer">
+                        <TextBlock Text="{x:Bind helper:Locale.Lang._OOBEStartUpMenu.CustomizationTitle}" FontSize="32" Margin="0,0,0,16"/>
+                        <controls:SettingsCard x:Name="SettingsCardItemLanguage"
+                                               Header="{x:Bind helper:Locale.Lang._OOBEStartUpMenu.CustomizationSettingsLanguageHeader}"
+                                               Description="{x:Bind helper:Locale.Lang._OOBEStartUpMenu.CustomizationSettingsLanguageDescription}"
+                                               HeaderIcon="{ui:FontIcon Glyph=&#xf57d;}"
+                                               Margin="0,2">
+                            <ComboBox x:Name="SelectLang"
+                                      SelectedIndex="{x:Bind SelectedLangIndex, Mode=TwoWay}"
+                                      HorizontalAlignment="Left" VerticalAlignment="Center"
+                                      ItemsSource="{x:Bind LangList}"
+                                      MaxDropDownHeight="200"
```

Comment:
We can remove the `DropDownOpened` and `DropDownClosed` events for there is no collision any more.